gitlab-pipeline/stage/release.yml

```
@@ -1,3 +1,12 @@
+variables:
+  IMAGE_DOCKER_VERSION:
+    value: "24.0.6"
+    description: |-
+      The version of the `docker` container image used for jobs.
+  IMAGE_DEBIAN_VERISON:
+    value: "buster"
```

Let's use numeric version rather than codename :pray: 
```suggestion
    value: "10"
```